src/components/app.tsx

```diff
@@ -40,9 +40,6 @@ export default function App() {
 			<HStack spacing="0" alignment="left" className="flex-grow">
 				<MainSidebar className="basis-52 flex-shrink-0 h-full" />
 				<main
-					style={ {
-						scrollbarGutter: 'stable',
-					} }
```

Comment:
`auto` is the default value of [scrollbarGutter](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter#values). With minimum height fix in https://github.com/Automattic/studio/pull/124, reserving scrollbarGutter space with `stable` should no longer be needed on Windows. macOS should not be affected.